### ISSUE TITLE
Include a small simple clap to build and wrap; run pluginval on the result

### ISSUE THREAD

**Issue by baconpaul** [free-audio/clap-wrapper#119]
1. Take a small simple clap. I'll probably use https://github.com/baconpaul/clap-c99-distortion
2. Make it a subdirectory (but not a sub-linked cmake) of clap-wrapper called "tests/test-clap" and make the cmake do the wrap and so on
3. Build it in CI
4. Have CI download https://github.com/Tracktion/pluginval for the OS in question and run it as a validation step on the resulting VST3, AUV2, etc

basically have CI test that we can build and wrap a micro clap
